apps/etterlatte-saksbehandling-ui/client/src/components/person/brev/NyttBrevModal.tsx

```diff
@@ -85,6 +85,24 @@ const SoeknadMottattDato = ({ control }: { control: Control<FilledFormData, any>
   />
 )
 
+const VedtakDato = ({ control }: { control: Control<FilledFormData, any> }) => (
+  <ControlledDatoVelger
+    name="mottattDato"
+    label="Når ble søknaden mottatt?"
+    control={control}
+    errorVedTomInput="Du må velge når søknaden ble mottatt"
+  />
+)
+
+const KlageMotattDato = ({ control }: { control: Control<FilledFormData, any> }) => (
+  <ControlledDatoVelger
+    name="mottattDato"
+    label="Når ble søknaden mottatt?"
+    control={control}
+    errorVedTomInput="Du må velge når søknaden ble mottatt"
```

Comment:
Inneholder ikke disse to akkurat de samme dataene?

Reply:
jo, takk takk fikset i https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/6592/commits/79ba11db76ecc0d7d136aa7ea985e64b2a8452ca